### DISCUSSION TITLE
Domain Connection Limits and Backoff Strategies

### DISCUSSION THREAD

**Discussion by ghost** [postalserver/postal#2931] [Help with using Postal]
I would like to ask if Postal supports the following (currently):
- Domain Connection Limits, that is to say, do not open more than 10 connections to each MX record for Yahoo at any given time, do not send more than 100 messages per-connection.
- Configurable Backoff Delay: If yahoo is just overwhelmed right now, and disconnects you with a soft fail (or just disconnects you, or times out) how long before it retries again, in succession.
- Mailing expiration, after a certain timeframe, the mailing stops since the send is no longer relevant.

**Comment by willpower232** [comment]
Domain Connection Limits - not a specific feature but I guess it depends how many workers you have running

Configurable Backoff Delay - there is a config file setting for the maximum number of attempts, there is already an increasing delay so it doesn't just fail immediately

Mailing expiration - not a feature at all, email sends pretty quickly depending on how many workers you have

**Reply by ghost** [reply]
Thank you!
